### ISSUE TITLE
Page title not descriptive uiowa a-z

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9275]
# Issue

SiteImprove is flagging the uiowa.edu a-z with a WCAG 2.1 A compliance issue.

Page title is not descriptive

The page title should clearly describe the content or purpose of the page. Siteimprove uses AI to review whether the title is descriptive. You can edit the review at any time if needed.

https://my2.siteimprove.com/Inspector/995976/A11y/Page?pageId=48524207626&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r114/failed/ckJ2enAxblQ=;MDc5NmM4MzE=

# Additional Info

Example page - https://uiowa.edu/a-z/f

SiteImprove browser plugin not reporting in Chrome

Page title - A-Z List | The University of Iowa | The University of Iowa

**Comment by james917** [comment]
This is in PR Ahoy for review.
